sdk/eventhubs/azure-messaging-eventhubs/src/test/java/com/azure/messaging/eventhubs/EventProcessorTest.java

```diff
@@ -293,7 +293,7 @@ public void testWithMultiplePartitions() throws Exception {
             .createConsumer(anyString(), eq("1"), any(EventPosition.class), any(EventHubConsumerOptions.class)))
             .thenReturn(consumer1);
         when(consumer1.receive()).thenReturn(
-            Mono.fromRunnable(() -> count.countDown()).thenMany(Flux.just(eventData1, eventData2)));
+            Mono.fromRunnable(count::countDown).thenMany(Flux.just(eventData1, eventData2)));
```

Comment:
I'm wary of this change. I've run across Zulu's JVM messing up during runtime and bombing out on these because it fails to interpret this method reference. I would advise to keep the changes limited to the scope of this PR.

Reply:
Same with the one below. I'll find the bug … 
The one on the left was causing the thing to fail compilation.
https://github.com/Azure/azure-sdk-for-java/pull/4038/commits/69577ac0aeaa0a6f2d658e57a832b493d9f6c167

https://bugs.openjdk.java.net/browse/JDK-8221420